### DISCUSSION TITLE
I have 2 question one is i want to download php 7.2 and want to in mongodb extenstion

### DISCUSSION THREAD

**Discussion by nouman6720** [leokhoa/laragon#856] [Show and tell]
I have 2 question one is i want to download php 7.2 and want to in mongodb extenstion

**Comment by luisAntonioLAGS** [comment]
I'll help you with the first one, I'll share the link to version 7.2 for x86 and x64
https://windows.php.net/downloads/releases/archives/php-7.2.34-Win32-VC15-x64.zip
https://windows.php.net/downloads/releases/archives/php-7.2.34-Win32-VC15-x86.zip

**Comment by Barrey** [comment]
> I have 2 question one is i want to download php 7.2 and want to in mongodb extenstion

First, you need to know which server you're using. If you're using Apache, download the TS (Thread Safe) version. Otherwise, use the NTS (Non-Thread Safe) version.
You also need to know your system specifications. For example, I'm using Windows with a 64-bit operating system. You can find this information by going to This PC -> Properties.

<img width="359" alt="image" src="https://github.com/user-attachments/assets/bbd07a96-a78f-42df-a000-98fa97c9a9f9">


you can download [here](https://windows.php.net/download) if you're using windows.

This also applies to MongoDB. Download the extension [here](https://pecl.php.net/package/mongodb), extract it, and move the .dll file (if you're using Windows) to the `ext` directory within your PHP version's folder.
Next, open `php.ini` (right-click on Laragon and find `php.ini`), then add the line `extension=php_mongodb`. Restart Laragon, and you can verify the installation by running `php -m`. Make sure MongoDB is listed.
